content/sensu-go/6.2/sensuctl/_index.md

```diff
@@ -26,11 +26,20 @@ sensuctl configure
 This starts the prompts for interactive sensuctl setup.
 When prompted, choose the authentication method you wish to use: username/password or OIDC.
 
+Upon successful authentication, sensuctl stores the access and refresh tokens in a "cluster" configuration file under the current user's home directory.
+For example, on Unix systems, sensuctl stores the tokens in `$HOME/.config/sensu/sensuctl/cluster`.
+
```

Comment:
I had another look at where we moved this -- on second thought it looks like this statement is coming too soon in the doc. We don't explain what access and fresh tokens are for another 20 lines, so I think a reader could reasonably be confused if they encounter this explanation here.

If we move this statement up as suggested, I think we need to move the rest of the explanation about access and refresh tokens up along with it.

Reply:
I added a suggestion (basically just moving the rest of the token explanation up here too)